src/manager.js

```diff
@@ -412,14 +451,19 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
 
     var slot = this.configureAd(thisEl);
 
-    if (typeof window.index_headertag_lightspeed === 'undefined') {
+    if (slot) {
+      slotsToLoad.push(slot);
+    }
+
+    if (typeof window.headertag === 'undefined' || window.headertag.apiReady !== true) {
       this.googletag.display(thisEl.id);
     } else {
-      window.index_headertag_lightspeed.slotDisplay(thisEl.id, ads);
+      window.headertag.display(thisEl.id);
```

Comment:
Do we need to confirm with IndexExchange that this will work with both the version of their header library on the Bulbs sites and the Kinja sites?